perf/src/lib.rs

```diff
@@ -52,6 +53,9 @@ pub struct CommonOpt {
     #[cfg(feature = "qlog")]
     #[clap(long = "qlog")]
     pub qlog_file: Option<PathBuf>,
+    /// disable ansi colors in logs
+    #[clap(long)]
+    pub no_color: bool,
```

Comment:
I wouldn't be surprised if there are environment variables that tracing-subscriber already knows about? Would be nicer than reinventing stuff here.